CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
 
 option(MBGL_WITH_CORE_ONLY "Build only the core bits, no platform code" OFF)
+option(MBGL_WITH_CLANG_TIDY "Build with clang-tidy checks enabled" OFF)
```

Comment:
Should we still use `MBGL_` prefix? :)

Reply:
For now, yes. We first need to cleanup the configuration a bit and then we may do one big rename.